### DISCUSSION TITLE
How to get started

### DISCUSSION THREAD

**Discussion by coding-cosmos** [eclipse/jkube#1874] [Q&A]
I know java . Now I want to have some practical experience that's why I want to contribute .
I have some questions:-
1) What are the prerequisite so that I can understand the codebase ?
2) In readme , here https://github.com/eclipse/jkube#getting-started  , some course is mentioned but I am not able to access it So how to access the course ?
3) What other languages or technologies I need to learn to get started with my contribution journey ?

**Comment by baruKreddy** [comment]
Hey @coding-cosmos , If this is your first time contributing to open Source and if you know basic java I'd suggest you to get started with first-timers-only labeled issues in the repo in which you would not need any other technologies for now. And for your other questions I'd let seniors comment on it.

**Reply by rohanKanojia** [reply]
I have a blogpost on how to get started with contributing to jkube as well https://rohaan.medium.com/how-to-contribute-to-eclipse-jkube-f3bc7f902bf4 . I hope that helps.
